Dockerfile.base

```diff
@@ -0,0 +1,35 @@
+FROM ubuntu:16.04
+
+RUN apt-get update -y -qq
+RUN apt-get install -y -qq \
```

Comment:
Are `libyaml-dev`, `libffi-dev`, and `libgdbm-dev` included as dependency? If not, please install.